coalahtml/helper.py

```diff
@@ -0,0 +1,95 @@
+import os
+import shutil
+
+from coalib.parsing.DefaultArgParser import default_arg_parser
+from coalahtml import Constants
+from coalahtml.tree import Tree
+
+
+def get_file(file_name: str, dir_path: str=os.path.dirname(__file__)):
```

Comment:
If you're using annotations you should use enoforce_signature probably. Also name is not very good.